src/ProjectTemplates/test/RazorPagesTemplateTest.cs

```diff
@@ -31,8 +33,6 @@ public async Task RazorPagesTemplate_NoAuthImplAsync()
             var createResult = await Project.RunDotNetNewAsync("razor");
             Assert.True(0 == createResult.ExitCode, ErrorMessages.GetFailedProcessMessage("razor", Project, createResult));
 
-            AssertFileExists(Project.TemplateOutputDir, "Pages/Shared/_LoginPartial.cshtml", false);
```

Comment:
Why did we remove this check?

Reply:
File existence is covered more thoroughly [here](https://github.com/aspnet/AspNetCore/blob/master/src/ProjectTemplates/test/BaselineTest.cs#L73)  so I figured it was best to only test it in one place.